src/main/java/org/jenkinsci/plugins/vstest_runner/VsTestBuilder.java

```diff
@@ -118,7 +120,7 @@
      * @param failBuild
      */
     @DataBoundConstructor
-    public VsTestBuilder(String vsTestName, String testFiles, String settings, String tests, String testCaseFilter, boolean enablecodecoverage, boolean inIsolation, boolean useVsixExtensions, String platform, String otherPlatform, String framework, String otherFramework, String logger, String otherLogger, String cmdLineArgs, boolean failBuild) {
+    public VsTestBuilder(String vsTestName, String testFiles, String settings, String tests, String testCaseFilter, boolean enablecodecoverage, boolean inIsolation, boolean useVsixExtensions, boolean useVs2017Plus, String platform, String otherPlatform, String framework, String otherFramework, String logger, String otherLogger, String cmdLineArgs, boolean failBuild) {

```

Comment:
Should probably just start using `@DataboundSetter` instead of this madness.

Reply:
Done.